api/services/events/v1/container.pb.go

```diff
@@ -114,6 +117,90 @@ func init() {
 	proto.RegisterType((*ContainerUpdate)(nil), "containerd.services.events.v1.ContainerUpdate")
 	proto.RegisterType((*ContainerDelete)(nil), "containerd.services.events.v1.ContainerDelete")
 }
+
+// Field returns the value for the given fieldpath as a string, if defined.
+// If the value is not defined, the second value will be false.
+func (m *ContainerCreate) Field(fieldpath []string) (string, bool) {
+	if len(fieldpath) == 0 {
+		return "", false
+	}
+
+	switch fieldpath[0] {
+	// unhandled: runtime
```

Comment:
Need to add a special case for:
```
	message Runtime {
		string name = 1;
		google.protobuf.Any options = 2;
	}
```

Or would defining it outside `ContainerCreate` do the trick?